### ISSUE TITLE
[Feature] Team names

### ISSUE THREAD

**Issue by ZeegCL** [alkarinv/BattleArena#103]
It would be nice if you can add team names, so the players can give unique names to their teams using a command like the actual "create" but with a little modification:

/team create <name> <player1> <player2> ... <playerN>

And the matches could be announced like:

Team foo[1250] vs Team bar[1250]

Regards.

**Comment by alkarinv** [comment]
Well, this isnt quite the same.. But inside of the teamConfig.yml, you can change the name of the default team names.

Otherwise, I don't see why they can't name themselves.. I'll put this as a low priority but I should be able to get it at some point.. just not sure how much it will be used.

**Comment by jkcgs** [comment]
Have you ever played WoW? In some servers, when players enter to some match or arena, they set a name for it, and that name is announced on the chat. I played on a server which the users set funny names, like TeamPotato. That can be the idea to set team names.

**Comment by alkarinv** [comment]
Let me see if I can make an option for it.  Also the names themselves would be temporary.  

So when I'm talking about team names I'm talking about /team create <player>.  Is that what you are talking about to?

**Comment by jkcgs** [comment]
Yep, it could be something like /team create [name], and the name would be optional
